templates/user-seed.conf.epp

```diff
@@ -1,4 +1,5 @@
-<%- | String $hash | -%>
+<%- | String $user,
+      String $hash | -%>
 [user_info]
-USERNAME=admin
+USERNAME=<%= $user %>
```

Comment:
thanks for the PR, can you please add a unit test for this change?

Reply:
The class had no test at all, so I kinda winged it, but, "added."